issuer.go

```diff
@@ -0,0 +1,111 @@
+package issuer
+
+import (
+	"bytes"
+	"compress/flate"
+	"crypto/ecdsa"
+	"encoding/json"
+	"github.com/skip2/go-qrcode"
+	"gopkg.in/square/go-jose.v2"
+	"strconv"
+	"time"
+)
```

Comment:
Can we organize these dependencies with `goimports`, so that the standard library packages are at the top?